checker/raw_result.go

```diff
@@ -256,3 +257,27 @@ type ReleaseAsset struct {
 	Name string
 	URL  string
 }
+
+// DangerousWorkflowData contains raw results
+// for dangerous workflow check.
+type DangerousWorkflowData struct {
+	ScriptInjections     []ScriptInjection
+	SecretInPullRequests []EncryptedSecret
+	UntrustedCheckouts   []UntrustedCheckout
+	// TODO: other
+}
+
+// UntrustedCheckout represents an untrusted checkout.
+type UntrustedCheckout struct {
+	File File
+}
+
+// ScriptInjection represents a script injection.
+type ScriptInjection struct {
+	File File
+}
+
+// EncryptedSecret represents an encrypted secret.
+type EncryptedSecret struct {
+	File File
+}
```

Comment:
Could we add more details to these structs like - the secret we think is at risk of leaking, job name, script injection value etc.?

Reply:
ah yes that slipped thru the crack. The value is already present in the `File.Snippet` (can it here instead), but job names, etc should be added. I'll do that, thanks

Reply:
Done:
```
"untrustedCheckouts": [
        {
          "file": {
            "snippet": "${{ github.event.pull_request.head.sha }}",
            "path": ".github/workflows/integration.yml",
            "offset": 45
          },
          "job": {
            "name": null,
            "id": "integration-trusted"
          }
        }
```